docs/conf.py

```diff
@@ -20,7 +19,16 @@
 html_title = "Determined AI Documentation"
 copyright = time.strftime("%Y, Determined AI")
 author = "ai-open-source@hpe.com"
-version = pathlib.Path(__file__).parents[1].joinpath("VERSION").read_text().strip()
+
+# Read the application version string from the VERSION environment
+# variable. Previously, this was read from a static VERSION file at the root of
+# the repository. Now, we read it from an environment variable set by a
+# Makefile, generated from version.sh in the repository root.
+version = os.environ.get("VERSION")
```

Comment:
just use `os.environ['VERSION']`

Reply:
I'd like to indicate explicitly to the user that they should set `VERSION` as an environment variable. I can raise an exception to halt execution, but I'd like to print a friendlier error message, too.

Reply:
that way it'll be obvious which error you're catching and you can print your message

Reply:
that way it'll be obvious which error you're catching and you can print your message

Reply:
I pushed some changes up for this. Let me know what you think!

Reply:
Good change but can you add it to all the other files?

Reply:
Good change but can you add it to all the other files?

Reply:
Absolutely.